src/app/components/ContactFieldProperty.js

```diff
@@ -10,28 +10,27 @@ import { getAllFieldLabels } from '../helpers/fields';
 const ContactFieldProperty = ({ field, value, uid, onChange, ...rest }) => {
     const handleChange = ({ target }) => onChange({ value: target.value, uid });
     const labels = getAllFieldLabels();
-    const val = Array.isArray(value) ? value[0] : value;
```

Comment:
Why is this there?

Reply:
Are you referring to the line `const val = Array.isArray(value) ? value[0] : value;`?

I guess it was there from the old logic. But that line is not there anymore

Reply:
I mean this kind of change should be in an other PR right?

Reply:
ouch, yeah, i branched from the wrong branch...